keyboards/sentraq/s60_x/default/config.h

```diff
@@ -1,20 +1,47 @@
-#ifndef DEFAULT_CONFIG_H
-#define DEFAULT_CONFIG_H
+#pragma once
 
 #include "config_common.h"
 
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Massdrop
```

Comment:
I realize this is left over from current master, but shouldn't this be

```suggestion
#define MANUFACTURER    Sentraq
```

?